datafusion/optimizer/src/analyzer/type_coercion.rs

```diff
@@ -982,6 +1056,128 @@ mod test {
         assert_analyzed_plan_eq(Arc::new(TypeCoercion::new()), plan, expected)
     }
 
+    #[test]
+    fn coerce_utf8view_output() -> Result<()> {
+        let expr = col("a");
+        let empty = empty_with_type(DataType::Utf8View);
+        let plan = LogicalPlan::Projection(Projection::try_new(
+            vec![expr.clone()],
+            Arc::clone(&empty),
+        )?);
+
+        // baseline
+        let expected = "Projection: a\n  EmptyRelation";
```

Comment:
It might be nice to create some sort of function / struct that avoids the repetition in the tests (and thus makes the intent of the test easier to understand

Maybe something like

```rust
        expect_coerced_plan(expected, plan, options)
```